internal/app/rds/rds.go

```diff
@@ -224,7 +229,7 @@ func (r *RDSFetcher) computeInstanceMetrics(dbInstance aws_rds_types.DBInstance,
 		MultiAZ:                    dbInstance.MultiAZ,
 		PendingMaintenanceAction:   pendingMaintenanceAction,
 		PendingModifiedValues:      pendingModifiedValues,
-		PerformanceInsightsEnabled: *dbInstance.PerformanceInsightsEnabled,
+		PerformanceInsightsEnabled: performanceInsightsEnabled,
```

Comment:
You can use [ToBool](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#ToBool) from aws sdk if you want here like so:
```go
PerformanceInsightsEnabled: aws.ToBool(dbInstance.PerformanceInsightsEnabled),
```

Instead of doing this [check](https://github.com/qonto/prometheus-rds-exporter/pull/12/files#diff-f6457e917cd954776edf471ccde9c797f09aeb4004ebfeec3440065409136458R206-R210)

Reply:
Good idea, I updated it with this method.